kroxylicious/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java

```diff
@@ -62,7 +62,7 @@ public void initChannel(SocketChannel ch) {
         if (logFrames) {
             pipeline.addLast("frameLogger", new LoggingHandler("io.kroxylicious.proxy.internal.DownstreamFrameLogger", LogLevel.INFO));
         }
-        
+
```

Comment:
It seems that the `mvn process-sources` command is removing trailing spaces for us. Is this something that perhaps could be added to the GitHub action that runs on each of these PRs?